VideoLocker/res/values/styles.xml

```diff
@@ -39,6 +39,10 @@
         <item name="android:buttonStyle">@style/edX.Widget.Button</item>
     </style>
 
+    <style name="CustomActionBarTheme.NoShadow" parent="CustomActionBarTheme">
```

Comment:
Since you're prefixing `CustomActionBarTheme.`, it should already implicitly inherit from `CustomActionBarTheme`; you shouldn't need to explicitly define it as the parent.